wiki/snapshot.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Snapshots
+layout: wiki
+---
+
+# Overview
+Snapshots are a snapshot of the chain at a particular blockheight.
+Gridcoin snapshots are created by trusted developers and they allow newer users to get into sync faster than syncing from 0.
+
+# Issues
+* Snapshots are not validated by the wallet.
+* Snapshots are an inherently centralised method of getting into sync.
+* Users may mistakenly install a malicious or corrupted snapshot. 
+If enough users install the bad snapshot then the integrity of the chain could be compromised.
+
+# Using snapshots
+## Starting from 5.0.0
```

Comment:
Since people aren't (or at least shouldn't be) running <5.0.0.0 anymore, I think we can just say something like "In the Wallet" instead 
```suggestion
## In the Wallet
```